examples/benchmarking/resnet50/README.md

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
I'm not sure the benchmarking utility is the right place to surface this work, if anything I'd expect our metrics API to log these metrics by default for end users

Reply:
There are 2 configs for this

1. Enable logging of these new metrics : Using ```profile: true``` in model-config.yaml
2. Enable reporting of these metrics in benchmarks. Using bechmakr config : ```handler_profiling: True```

We can turn on 1) by default if there is consensus.

Reply:
I suggest making profiling as option in benchmark, not default since traffic volume in  benchmark is high and record each request profiling record is expensive. it would decrease perf result.

Reply:
I suggest Resnet-50 benchmark result in private doc and share it with internal.  You. can add resnet-50 in example dir.

Reply:
Done. Removed the report